tests/test.mk

```diff
@@ -1,9 +1,14 @@
 TEST_SRC = $(wildcard tests/test_*.cc)
 TEST = $(patsubst tests/test_%.cc, tests/test_%, $(TEST_SRC))
 
+STATIC_LIBS = build/libps.a
+ifeq ($(USE_TP), 1)
+STATIC_LIBS += $(TP_INSTALL_PATH)/lib64/libtensorpipe.a $(TP_INSTALL_PATH)/lib64/libtensorpipe_uv.a
```

Comment:
Would you also add tensorpipe as a submodule? thanks

Reply:
Also better to automatically build libtensorpipe.a in pslite makefile.

Reply:
@ymjiang  I tried to add TP into deps.mk, but it appears to encounter some wired issues when make -j (but make by only one thread seems to work fine). Specifically, the build process terminates after successfully build TP (ps-lite is not being build). So I suggest to build TP as a third party dependency outside ps-lite (similar to UCX).

Add \$ (TP) after \$ (ZMQ) in Makefile, and  in deps.mk:
```
TP = ${DEPS_PATH}/include/tensorpipe/tensorpipe.h
${TP}:
    git clone TP and build, as shown in this comment https://github.com/bytedance/ps-lite/pull/111#issuecomment-1380879514.
```

Reply:
OK, then let's consider fixing this in later MRs. Thank you for you contribution!